frontend/src/components/phones/onboarding/PurchasePhonesPlan.test.tsx

```diff
@@ -33,50 +33,64 @@ jest.mock("../../../functions/getPlan", () => {
   };
 });
 
+const escapeRe = (s: string) => s.replace(/[.*+?^${}()|[\]\\]/g, "\\$&");
+const byMsgId = (id: string) => new RegExp(`\\[${escapeRe(id)}\\]`);
```

Comment:
questions (blocking): Why are we defining these `escapeRe` and `byMsgId` helpers in 5 separate files rather than defining them in one place and importing them?